_posts/2017/2017-03-10-creating-a-sales-analytics-app-with-ruby-on-rails-2.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Creating a Sales Analytics App with Ruby on Rails, part 2"
+date: "2017-03-10 07:00:00"
```

Comment:
Post is scheduled for 2017-04-04. Please change in layout and filename accordingly.